### ISSUE TITLE
Screen Reader speech does not differentiate individual Note Anchored Lines

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32297]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

It is not possible with a screen reader to know where note anchored lines start and stop.  If there are two or more and some cross over the blind user can’t tell what the diagram is showing.
STEPS:
With a screen reader on use the right arrow to navigate across the notes in bar 1 and then across bar 2.  Notice that the speech output is the same for both.  Both say “Start of Note Anchored Line” on the C & D and “End of Note Anchored line” on the E & F. 

For those who can’t see the print music attached.
Bar 1 line goes from C to F and another from D to E.
Bar 2 lines go from C to E and D to F.

This could be even more complicated if some lines are attached vertically from one staff to another close to others that are horizontal or cross over.

Suggest these need to announce when you Alt arrow on to them what bar/beat/staff they start and end on.  This has been suggested for other lines also.  It is too much info to have in the general speech string as you arrow along, so Alt Arrow would be better when on the line itself.  some sort of shortcut that gives additional info when on an element has also been suggested and if this is implemented this would be an excellent case for that.

### Supporting files, videos and screenshots

[Note anchored lines speech output.zip](https://github.com/user-attachments/files/25435943/Note.anchored.lines.speech.output.zip)

### In which versions of MuseScore Studio is this issue present?

4.6

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
